src/maintainer/infrastructure.rst

```diff
@@ -285,8 +285,14 @@ here `<https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/
 GitHub Actions
 --------------
 
-We use GitHub actions to rerender feedstocks and also run our pull request automerge service. We do not currently support builds on
-GitHub Actions.
+We use GitHub actions to rerender feedstocks and also run our pull request automerge service.
+We also support builds on GitHub Actions.
+
+Woodpecker
+----------
+
+We support `Woodpecker <https://github.com/woodpecker-ci/woodpecker>`__ for Linux ARMv8 builds.
```

Comment:
Woodpecker is only the project that orchestrates CI workflows, I think. There's no hosted service running on their end; that needs to be provided by the user (us). I think we used Woodpecker at some point to run jobs on a donated machine, but I am not sure this is still running.